src/main/js/field/input/NumberInput.js

```diff
@@ -24,13 +24,13 @@ NumberInput.propTypes = {
     readOnly: PropTypes.bool,
     inputClass: PropTypes.string,
     name: PropTypes.string,
-    value: PropTypes.number,
+    value: PropTypes.string,
```

Comment:
Will this affect the way the field gets displayed?

Reply:
From what I've seen so far, no.